device.yml

```diff
@@ -325,18 +325,21 @@ registers:
     access: Write
     maskType: Channel3RangeConfig
     description: Selects the flow range for the channel 3.
-  Reserved0: &reserved
+  TemperatureValue: 
     address: 92
     type: U8
     access: Read
-    description: Reserved for future use.
-    visibility: private
-  Reserved1:
-    <<: *reserved
+    description: Temperature sensor reading value.
+  EnableTemperatureCalibration:
     address: 93
-  Reserved2:
-    <<: *reserved
+    type: U8
+    access: Read
+    description: Enable flow adjustment based on the temperature calibration.
+  UserTemperatureCalibration:
```

Comment:
What is the difference between `TemperatureValue` and `UserTemperatureCalibration`? Is this a reference value used to establish the adjustment automatically?

I am not very fond of adding the general term `User` here, since a "user" is very vaguely defined (and could be automated away in the future). It would be more helpful to describe here what exactly is expected this value to contain, e.g. if this is a reference value, we might consider changing this to `ReferenceTemperatureValue` and the description to more explicitly indicate what is the meaning of the values provided here.

Reply:
`TemperatureValue` is the actual value of the device temperature
`UserTemperatureCalibration` is used for calibration purposes only if the user decides to calibrate the device. This temperature calibration value is the value at which the calibration was done.

`User` is used to be consistent with former register description used for the calibration of the device, when performed by the user (the device always contains the factory calibration values, so `User` is used to distinguish this).

Reply:
`TemperatureCalibrationValue` would be more clear?

Reply:
> TemperatureCalibrationValue would be more clear?

Definitely, that would be great if it works!

Reply:
It should be noted that adding the word User would make some sense given the current interface (https://github.com/harp-tech/device.olfactometer/blob/621022cf845f293f16a8558483cdf3ec015400f2/device.yml#L38).

Should we break this while we are still in 0.x?

Reply:
In fact, I've double checked and this register is shared for both the user and the factory calibration, so in the end `TemperatureCalibrationValue` seems more appropriated.  Do you agree @bruno-f-cruz , so that I can make the mods and merge these files?